.github/vale/Vocab/Industry/accept.txt

```diff
@@ -1,6 +1,7 @@
 AWS
 Amazon
 Apple
+Azure( Blob Storage)?
```

Comment:
Is the ` ( Blob storage)?` intentional?

Reply:
yeah it's an optional (acceptable) suffix to Azure, so it approves both Azure and Azure Blob Storage. We have the same for `GitHub( Actions)?`

Reply:
Makes sense. Thanks